utils/sb2

```diff
@@ -448,17 +448,14 @@ function sboxify_environment()
 	SBOX_TARGET_TOOLCHAIN_DIR=$(dirname "$SBOX_CROSS_GCC_DIR")
 
 	if [ -z "$SBOX_MODE_PATH" ]; then
-		# /sb2/wrappers and /sb2/scripts contain utilities for
-		# development support, and are available in the
-		# software development modes ("simple" and "devel")
-	    TMP_PATH=/sb2/wrappers:/sb2/scripts:$HOME/.scratchbox2/$SBOX_TARGET/bin
-	    if [ ! -z "$CCACHE_TOOLS_PATH" ]; then
-		TMP_PATH=$TMP_PATH:$CCACHE_TOOLS_PATH
-	    fi
-	    export PATH=$TMP_PATH:$SBOX_DIR/bin:$PATH:/sbin:/usr/sbin:$SBOX_TARGET_ROOT/bin:$SBOX_TARGET_ROOT/usr/bin:$SBOX_TARGET_ROOT/usr/local/bin
+		new_PATH=$HOME/.scratchbox2/$SBOX_TARGET/bin
+		new_PATH+=:${CCACHE_TOOLS_PATH+$CCACHE_TOOLS_PATH:}$SBOX_DIR/bin:$PATH
+		PATH=$new_PATH:/sbin:/usr/sbin:$SBOX_TARGET_ROOT/bin
+		PATH+=$SBOX_TARGET_ROOT/usr/bin:$SBOX_TARGET_ROOT/usr/local/bin
```

Comment:
Leading colon is missing here :( As a result, `$SBOX_TARGET_ROOT/bin` and `$SBOX_TARGET_ROOT/usr/bin` are essentially missing from PATH, as in

```
PATH=/home/mersdk/.scratchbox2/SailfishOS-4.4.0.58-i486.default/bin:/usr/bin:/usr/local/bin:/bin:/usr/bin:/sbin:/usr/sbin:/srv/mer/targets/SailfishOS-4.4.0.58-i486.default/bin/srv/mer/targets/SailfishOS-4.4.0.58-i486.default/usr/bin:/srv/mer/targets/SailfishOS-4.4.0.58-i486.default/usr/local/bin
```

Reply:
#12